util/devel/greptestopts

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+if [ ! -z "$CHPL_HOME" ] ; then
+    chapelDir="$CHPL_HOME"
+else
+    chapelDir="./"
+fi
+
+pushd $chapelDir > /dev/null
+
+find test -wholename "*/.svn" -prune -o \( -name \*.compopts -o -name COMPOPTS -o -name \*.execopts  -o -name EXECOPTS -o -name \*.perfcompopts -o -name PERFCOMPOPTS -o -name \*.perfexecopts -o -name PERFEXECOPTS \) -print0 | xargs -0 grep -H "$@"
```

Comment:
There are ml* and cc* versions of these for multilocale and commcounts testing.

I see these:

```
[I] ➜ find test -name "*[^.]compopts" | rev | cut -d"." -f1 | rev | sort | uniq
cc-compopts
lastcompopts
ml-compopts
perfcompopts
```

Reply:
(and same for execopts versions and uppercase versions)